### PR TITLE
refactor: adjust booster lengths

### PR DESCRIPTION
**Pull request by Psychotic10**
Joe and i think that some of these times needa a little more but not much

### CODE REVIEW

File: data/items.json [hunk 1]
```diff
@@ -421,7 +421,7 @@
         "boosterEffect": {
             "boosts": ["multi"],
             "effect": 1,
-            "time": 30
+            "time": 55
```
Comment (tekoh) [inline data/items.json:424]: cookies are already everywhere. no.

File: data/items.json [hunk 2]
```diff
@@ -437,8 +437,8 @@
         "max": 3,
         "boosterEffect": {
             "boosts": ["xp"],
-            "effect": 0.2,
-            "time": 300
+            "effect": 0.5,
+            "time": 600
```
Comment (tekoh) [inline data/items.json:441]: no.

File: data/items.json [hunk 3]
```diff
@@ -741,7 +741,7 @@
         "boosterEffect": {
             "boosts": ["cooldown"],
             "effect": 0.5,
-            "time": 450
+            "time": 600
```
Comment (tekoh) [inline data/items.json:744]: too much

File: data/items.json [hunk 4]
```diff
@@ -909,7 +909,7 @@
         "boosterEffect": {
             "boosts": ["fight"],
             "effect": "improves fight strength",
-            "time": 3600
+            "time": 2700
```
Comment (tekoh) [inline data/items.json:912]: i like this

**Review by tekoh — CHANGES_REQUESTED** [review]

File: data/items.json (hunk 1 above)
Comment (Psychotic10) [reply data/items.json:424]: Then what is a good time?
565DAB5C-80F5-46C3-9149-316B457EFD87

File: data/items.json (hunk 2 above)
Comment (Psychotic10) [reply data/items.json:441]: Ok what about not changing the effect but the time?

File: data/items.json (hunk 4 above)
Comment (Psychotic10) [reply data/items.json:912]: Ok

File: data/items.json (hunk 3 above)
Comment (Psychotic10) [reply data/items.json:744]: How long would you say is enough?

File: data/items.json (hunk 1 above)
Comment (tekoh) [reply data/items.json:424]: i think 30 seconds is suitable

File: data/items.json (hunk 3 above)
Comment (tekoh) [reply data/items.json:744]: 450

File: data/items.json (hunk 1 above)
Comment (Psychotic10) [reply data/items.json:424]: What about 40?

File: data/items.json (hunk 3 above)
Comment (Psychotic10) [reply data/items.json:744]: Bruh

File: data/items.json (hunk 1 above)
Comment (Psychotic10) [reply data/items.json:424]: Also think about the cd of typing $use cookie and then having a cd so u can’t stack to 7 and then it is over already

**Pull request closed without merging**